Docs/ProjectSpec.md

```diff
@@ -897,12 +909,17 @@ schemes:
       coverageTargets:
         - MyTarget1
         - ExternalTarget/OtherTarget1
+        - package: LocalPackage/TestTarget
       targets: 
         - Tester1 
         - name: Tester2
           parallelizable: true
           randomExecutionOrder: true
           skippedTests: [Test/testExample()]
+        - target: 
+          package: APIClient/APIClientTests
```

Comment:
```suggestion
              package: APIClient/APIClientTests
```

Reply:
This is not really consistent in the way the current yml is constructed. Is there a specific reason why this is indented here?